pkg/grpc/server.go

```diff
@@ -55,6 +55,7 @@ func init() {
 var (
 	log = logging.LoggerForModule()
 
+	maxMsgSizeSetting         = env.RegisterIntegerSetting("ROX_GRPC_MAX_MESSAGE_SIZE", maxMsgSize)
```

Comment:
Shall we start calling it a default?

```suggestion
	maxMsgSizeSetting         = env.RegisterIntegerSetting("ROX_GRPC_MAX_MESSAGE_SIZE", defaultMaxMsgSize)
```